doc/toolbox-create.1.md

```diff
@@ -4,48 +4,144 @@
+  needs to be run as `root`
+
+Several `--volume` options to mount paths from the host filesystem (more about
+these in the next section).
+
+### Important mounts of a toolbox container
+
+Toolbox containers use Toolbox itself as it's entry-point (`toolbox
+init-container`). This requires the binary to be present inside of the
+container. If the file is deleted on the host then the container **will stop**
+working.
+
+The binary, using which `toolbox create` was executed, is mounted inside of the
+container.
+
+- `--volume path/to/toolbox/binary:/usr/bin/toolbox:ro`
+
+Current user's home directory is mounted inside of the container. If the home
+directory is a symlink (as is the case in Fedora Silverblue where the `/home`
+directory is a symlink to `/var/home`) then the path canonicalized and used also
+in the container.
+
+- `--volume /home/<username>:/home/<username>:rslave`
+- `--volume /var/home/<username>:/var/home/<username>:rslave`
+
+Several root directories of the host filesystem are mounted in `/run/host`
+inside of a toolbox container to provide as much flexibility.
+
+- `--volume /etc:/run/host/etc`
+- `--volume /dev:/run/host/dev`
+- `--volume /media:/media:rslave` or `--volume /run/media:/run/media:rslave`
+- `--volume /mnt:/mnt:rslave`
+- `--volume /run:/run/host/run:rslave`
+- `--volume /tmp:/run/host/tmp:rslave`
+- `--volume /usr:/run/host/usr:rslave` + `rw` if `/usr` is
+  writeable or `ro` if `/usr` is not writeable
+- `--volume /var:/run/host/var:rslave`
+
+Toolbox uses `flatpak-session-helper`[0] for tracking some of the key host files
+(`/etc/localtime` and `/etc/timezone`).
+
+- `--volume /run/user/1000/.flatpak-helper/monitor:/run/host/monitor` (this path
+  is just an example one, the number depends on the user's UID)
+
+> NOTE: The `flatpak-session-helper` will be removed/replaced in the future
+> because it is called using session bus which is not possible when Toolbox is
+> ran as root.
+
+The host's DBus system socket is mounted inside of a toolbox container.
 
-The following options are understood:
+- `--volume /run/dbus/system_bus_socket:/run/dbus/system_bus_socket`
 
-**--candidate-registry**
+The host's `XDG_RUNTIME_DIR` is mounted inside of a toolbox container to have
+access to the same runtime directory.
 
-Pull the base image from `candidate-registry.fedoraproject.org`. This is
-useful for testing newly built images before they have moved to the stable
-registry at `registry.fedoraproject.org`.
+- `--volume /run/user/1000:/run/user/1000` (this
+  path is just an example one, the number depends on the user's UID)
+
+If a KCM (SSSD Kerberos Cache Manager) socket is detected on the host, it is
+mounted inside of the container automatically. Thanks to this, authentication
+using Kerberos will work even in a container.
+
+- `--volume /run/.heim_org.h5l.kcm-socket:/run/.heim_org.h5l.kcm-socket` (this
+  path is just an example one, the socket may be in a different location)
+
+### Entry-point of toolbox containers
```

Comment:
Yes, this is a good idea. I split this out into https://github.com/containers/toolbox/pull/677